packages/gatsby-remark-embed-snippet/README.md

```diff
@@ -332,3 +270,119 @@ function App() {
   )
 }
 ```
+
+### Specifying snippets by name
+
+As an alternative to selecting a range of lines from a file, you can add `start-snippet{snippet-name}` and `end-snippet{snippet-name}` in comments in your files. The inclusion of a name for a snippet allows you to create an example file that contains multiple snippets that you reference from different places.
+
+You specify that you want to only include a named snippet from the embed using the syntax `{snippet: "snippet-name"}`.
+
+**Rust example**:
```

Comment:
Do you think we can get a JavaScript example of specific snippets by name? Rust is good but most off our users would relate more to a JS code snippets.

Reply:
I added a Rust example in part to show that `gatsby-remark-embed-snippet` now understands `.rs` as a Rust file extension, but that might have not been obvious.